storage/iam.rb

```diff
@@ -109,6 +109,46 @@ def add_bucket_conditional_iam_binding bucket_name:, role:, member:, title:, des
   # [END storage_add_bucket_conditional_iam_binding]
 end
 
+def remove_bucket_conditional_iam_binding bucket_name:, role:, title:, description:, expression:
+  # [START storage_remove_bucket_conditional_iam_binding]
+  # bucket_name = "Your Google Cloud Storage bucket name"
+  # role        = "Bucket-level IAM role"
+  # title       = "Condition Title"
+  # description = "Condition Description"
+  # expression  = "Condition Expression"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new
+  bucket = storage.bucket bucket_name
+
+  bucket.policy requested_policy_version: 3 do |policy|
+    policy.version = 3
```

Comment:
Given that there should be an existing binding with a condition, does `version` need to be set here? In the google-cloud-storage [`Bucket#update_policy` example](https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-storage/lib/google/cloud/storage/bucket.rb#L1978), `version` is not set but is shown as already being `3`. Is that example incorrect? (I can investigate.)

Reply:
It's best practices to always set version to 3 given the API design. 

Probably should update that in documentation.

Reply:
Thanks, will do!